src/nginxconfig/generators/conf/security.conf.js

```diff
@@ -58,13 +58,186 @@ export default (domains, global) => {
         config.push(['location /security.txt', {
             return: '301 /.well-known/security.txt',
         }]);
-        
+
         // Custom security.txt path
         config.push(['location = /.well-known/security.txt', {
             alias: `${global.security.securityTxtPath.value}`,
         }]);
     }
 
+    if (global.security.blockCommonExploits.computed) {
```

Comment:
```suggestion
    if (global.security.blockCommonExploits.computed) {
      // Based on https://github.com/NginxProxyManager/nginx-proxy-manager/blob/v2.9.0/docker/rootfs/etc/nginx/conf.d/include/block-exploits.conf
```
Let's leave a comment for where this was sourced from